src/Listener/JsonApiListener.php

```diff
@@ -483,6 +483,12 @@ protected function _getEntityList($entityName, $associations)
      */
     protected function _checkRequestData()
     {
+        // prevent false positives for GET requests using query parameters
+        // as those will also have request data
```

Comment:
mr. @davidyell, I completely misread your comment. Somehow my brain did not pick up the plugin. It seems you were spot on, it is indeed the PrgComponent. I am updating my comment in the code. Thanks!